caf-kubernetes-client/src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,4229 @@
+#
```

Comment:
This swagger.json will need extended with the calls + objects that the Autoscaler requires, but I will do a separate PR for that when I update the Autoscaler to use this new repo.

Reply:
I added the swagger.json as we only need a subset of the full api available at:

https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json

However, even with only one endpoint added, this swagger.json is already fairly big, because of all the model objects that needed to be added as well, and will increase a lot when I do the autoscaler (it has 5 additional endpoints).

Maybe the way I had it originally would be less maintenance?, i.e. remove this swagger.json and just update the OpenAPI plugin to generate the client from the full file:

```
<inputSpec>https://raw.githubusercontent.com/kubernetes/kubernetes/${kubernetesOpenApiSpecVersion}/api/openapi-spec/swagger.json</inputSpec>
```

The disadvantage of that is we would be generating classes that are not used, but the advantage would be less maintenance in trying to keep this file up to date?

Reply:
Ok given the size let's just pull the api as you suggest.